static/js/components/StudioList.tsx

```diff
@@ -63,10 +64,16 @@ export function StudioListItem(props: ListItemProps): JSX.Element {
   )
 
   return (
-    <li className="py-2" onClick={onClick}>
```

Comment:
Switched from padding to margin so that the space between items would not be clickable:

**Before / After**

<img width="273" alt="between_items_before" src="https://user-images.githubusercontent.com/9010790/151624935-66b6c6af-db5e-46f8-9259-f99a17cebc21.png"><img width="377" alt="between_items_now" src="https://user-images.githubusercontent.com/9010790/151624937-e04d8564-10e8-4a25-8bec-6ee5dcdc4cbc.png">

The margin is twice as big since these adjacent margins are collapsed.